.github/workflows/linux.yml

```diff
@@ -348,11 +349,13 @@ jobs:
           python -m pip wheel -v --no-deps --wheel-dir ${BUILD_DIR} \
               --config-settings=override=cross.arch="manylinux_2_31_x86_64" \
               --config-settings=override=cmake.options.BUILD_FAST_TOKENIZERS="${{ matrix.build_fast_tokenizers }}" \
-              --find-links ${INSTALL_DIR}/tools \
+              ${{ needs.openvino_download.outputs.ov_wheel_source || env.DEFAULT_OV_WHEEL_SOURCE }} \
               ${OPENVINO_TOKENIZERS_REPO}
         env:
           CMAKE_BUILD_PARALLEL_LEVEL: '4'
           CMAKE_GENERATOR: 'Unix Makefiles'
+          DEFAULT_OV_WHEEL_SOURCE: "--find-links ${{ env.INSTALL_DIR }}/tools"
```

Comment:
lets leave argument --find-links as part of "python -m pip wheel" command and substitute it's value only

Reply:
We can't, because when using nightly/rc/release packages we need --extra-index-url, not --find-links